packages/devtools_testing/lib/support/performance_test_data.dart

```diff
@@ -71,6 +71,30 @@ final jankyFrame = FlutterFrame('jankyFrame')
   ..eventFlows[1] = (goldenRasterTimelineEvent.deepCopy()
     ..time = (TimeRange()
       ..start = const Duration(milliseconds: 68)
+      ..end = const Duration(milliseconds: 88)));
+
+final jankyFrameUiOnly = FlutterFrame('jankyFrameUiOnly')
+// ignore: invalid_use_of_visible_for_testing_member
```

Comment:
Curious why this is required.

Reply:
I think it's because I'm using a visibleForTesting method `deepCopy` from a different package. The test data is in devtools_testing but the model class is in devtools_app. I would be in favor of moving all the devtools_testing code back into devtools_app - I don't think it was ever used as intended. @jakemac53 do you recall why it was moved into a separate package to begin with and if it is still useful to be kept separate?

Reply:
The [original PR](https://github.com/flutter/devtools/pull/1035) has a pretty thorough explanation. There was a goal to share integration tests between devtools and webdev it sounds like? I have no idea if this is still relevant.

Reply:
Given the original PR was almost 2 years ago and devtools_app is still the only package that depends on devtools_testing, I think we could simplify things by bringing the devtools_testing code back into devtools_app. Plus, once we are no longer shipping on pub, I don't know that the devtools_testing package will have any use anyway @jacob314 @grouma thoughts?

Reply:
Sounds good to me.